coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/RelToTrinoConverter.java

```diff
@@ -76,8 +77,25 @@ public RelToTrinoConverter(Map<String, Boolean> configs) {
    * @return SQL string
    */
   public String convert(RelNode relNode) {
-    RelNode rel = convertRel(relNode, configs);
-    return convertToSqlNode(rel).accept(new TrinoSqlRewriter()).toSqlString(TrinoSqlDialect.INSTANCE).toString();
+    return convertDash(relNode);
+
+    //            RelNode rel = convertRel(relNode, configs);
+    //            SqlNode oldSqlNode =  convertToSqlNode(rel);
+    //            return oldSqlNode.accept(new TrinoSqlRewriter()).toSqlString(TrinoSqlDialect.INSTANCE).toString();
+  }
+
+  /**
+   * Temporary method to enable translations via CoralSqlNodeToTrinoSqlNodeConverter
+   */
+  public String convertDash(RelNode relNode) {
```

Comment:
Could we give this method a more descriptive name? Not following what `convertDash` means.